app/services/container_dashboard_service.rb

```diff
@@ -20,8 +20,12 @@ def all_data
   end
 
   def status
-    if @ems.present? && @ems.kind_of?(ManageIQ::Providers::Openshift::ContainerManager)
-      routes_count = @ems.container_routes.count
+    if @ems.present?
+      if @ems.kind_of?(ManageIQ::Providers::Kubernetes::ContainerManager)
```

Comment:
I think that the check @ems.respond_to?(:container_routes) would be better here, because it is not dependent strictly on k8s, and will work with any other future provider types which have (or have no) routes.